### PR TITLE
Update to ASDK v6.10.0

### PR DESCRIPTION
**Pull request by mohsenoid**

### CODE REVIEW

File: compose/app/src/main/java/com/firework/example/compose/ExampleApplication.kt [hunk 1]
```diff
@@ -14,7 +14,6 @@ class ExampleApplication : Application() {
         // build Firework Android SDK v6 configuration
         val config =
             FireworkSdkConfig.Builder(this)
-                .checksumRequired(false)
```
Comment (mohsenoid) [inline compose/app/src/main/java/com/firework/example/compose/ExampleApplication.kt:17]: Deprecated

File: compose/build.gradle.kts [hunk 2]
```diff
@@ -1,12 +1,11 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "8.2.2" apply false
-    id("com.android.library") version "8.2.2" apply false
```
Comment (mohsenoid) [inline compose/build.gradle.kts:4]: Not needed

File: compose/build.gradle.kts [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+    id("com.android.application") version "8.3.1" apply false
```
Comment (mohsenoid) [inline compose/build.gradle.kts:3]: Updated to the latest

File: compose/gradle.properties [hunk 4]
```diff
@@ -21,5 +21,4 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
-android.defaults.buildfeatures.buildconfig=true
```
Comment (mohsenoid) [inline compose/gradle.properties:24]: Deprecated and causes a warning

File: compose/gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```
Comment (mohsenoid) [inline compose/gradle/wrapper/gradle-wrapper.properties:3]: Updated by AGP

File: compose/settings.gradle.kts [hunk 6]
```diff
@@ -11,6 +11,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
+        mavenLocal()
```
Comment (mohsenoid) [inline compose/settings.gradle.kts:14]: Nice to have for testing before release

**Review by OstapOpalynskyi-fw — APPROVED** [review]

File: single_host_livestream/app/build.gradle.kts [hunk 7]
```diff
@@ -46,10 +46,15 @@ android {
         abortOnError = true
         ignoreWarnings = false
         warningsAsErrors = true
+
+        disable.apply {
+            add("LeanbackUsesWifi") // required for ivs-player-1.18.0 AndroidManifest ACCESS_WIFI_STATE permission
```
Comment (mohsenoid) [inline single_host_livestream/app/build.gradle.kts:51]: image

**Pull request merged**